pages/web-sites.md

```diff
@@ -120,8 +120,14 @@ For other sites:
 
 To create a tag, use `npm version [major | minor | patch]`.
 
-Afterwards, make sure to push both version change commit and the tag:
-`git push --tags origin master`
+Afterwards, make sure to push both version change commit and the tag to the main jQuery repo.
+
+<div class="warning">You'll want to check your remote setup using ```git remote -v``` and check which remote is used for the main jQuery repo.</div>
```

Comment:
well, this is how it looks on GitHub:
![contribute_jquery_org_web-sites_md_at_clarifydeploy_ _agcolom_contribute_jquery_org](https://cloud.githubusercontent.com/assets/6025224/7444461/71229b7e-f184-11e4-92cc-907ab95030b1.png)

If it works fine for our sites that's in fact a buggy behaviour of our markdown parser so I'd prefer to fix it (just use html `<code>` syntax directly)

-

Oh and yeah, except for this line, let's be consistent and use a single backtick for inline code instead.